common/named_colors/formable_MolonLabe_colors.txt

```diff
@@ -197,7 +197,7 @@ colors = {
 
 	liguria_color = rgb { 157 75 26 }
 	
-	Cimmeria_color = rgb { 173 0 40 }
+	Cimmeria_color = rgb { 13 252 148 }

```

Comment:
would love to see these properly cased to reduce likelihood of errors

Reply:
There is no errors they work fine in game